src/modules/content-tree/components/list-item/list.item.component.js

```diff
@@ -23,44 +75,70 @@ class ListItem extends Component {
      */
     renderIcon() {
         const { contentTypeIdentifier, selected } = this.props;
-        const customPath =
-            eZ.helpers.contentType.getContentTypeIconUrl(contentTypeIdentifier) || eZ.helpers.contentType.getContentTypeIconUrl('file');
         const iconAttrs = {
-            customPath,
             extraClasses: `ez-icon--small ez-icon--${selected ? 'light' : 'dark'}`,
         };
 
+        if (!this.state.isLoading) {
+            iconAttrs.customPath =
+                eZ.helpers.contentType.getContentTypeIconUrl(contentTypeIdentifier) || eZ.helpers.contentType.getContentTypeIconUrl('file');
+        } else {
+            iconAttrs.name = 'spinner';
+            iconAttrs.extraClasses = `${iconAttrs.extraClasses} ez-spin`;
+        }
+
         return (
             <span className="c-list-item__icon">
                 <Icon {...iconAttrs} />
             </span>
         );
     }
 
+    renderLoadMoreBtn() {
+        let loadingSpinner = null;
+
+        if (!this.state.isExpanded || !this.checkCanLoadMore()) {
+            return null;
+        }
+
+        if (this.state.isLoading) {
+            loadingSpinner = <Icon name="spinner" extraClasses="ez-spin ez-icon--small ez-icon--dark c-list-item__load-more-btn-spinner" />;
+        }
+
+        const loadMore = Translator.trans(/*@Desc("Load More")*/ 'content_tree_list.item.load_more', {}, 'universal_discovery_widget');
```

Comment:
Are we OK with `universal_discovery_widget` domain for this translation? ping @dew326

Reply:
Nope, we should have a new domain.